guides/development-workflow.md

```diff
@@ -190,6 +190,7 @@ To deploy:
   ```
   git fetch
   git checkout master
+  git pull
```

Comment:
@tomdxw I would be fine putting that into the dev guide:

```
git checkout master
git pull --ff-only
```

@tahb what do you think?

I can't imagine why anyone would _pull_ with `--no-ff`. That would be saying "I want a record in the main history of the project that I pulled this branch down onto my machine".

An alternative approach which bypasses some of the concerns would be:

```
git fetch origin
git checkout master
git reset --hard origin/master
```

That says "ditch whatever changes I have to local master: just point it at whatever origin is pointing at"

But in the (rare, probably not worth accounting for) case that someone has accidentally committed something to local master, this would blow those changes away and it would be reflog spelunking time. A plus of `pull --ff-only` is that it _errors_ when something has gone wrong.